packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -217,7 +217,7 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
         const styles: React.CSSProperties = {};
 
         styles.minWidth = this.props.collapsed
-            ? this.props.collapsedWidth
+            ? toPx(this.props.collapsedWidth)
```

Comment:
Doesn't react convert this for us?

Reply:
If it does we should remove toPx from the minWidth value below. I found this writing tests where I'm getting a number back as opposed to a string. I'm fine either way; just think we should be consistent with our application here.